classnames/classnames.d.ts

```diff
@@ -6,7 +6,7 @@
 declare type ClassValue = string | number | ClassDictionary | ClassArray;
 
 interface ClassDictionary {
-	[id: string]: boolean;
+	[id: string]: any;
```

Comment:
sorry, I am failing at sharing from the playground today :)

screencap:

![image](https://cloud.githubusercontent.com/assets/723999/18796443/4849fc3e-8197-11e6-8bc2-b7bdc6cd1fac.png)

[link to a working playground](https://www.typescriptlang.org/play/index.html#src=declare%20type%20ClassValue%20%3D%20string%20%7C%20number%20%7C%20ClassDictionary%20%7C%20ClassArray%3B%0D%0A%0D%0Ainterface%20ClassDictionary%20%7B%0D%0A%09%5Bid%3A%20string%5D%3A%20boolean%3B%0D%0A%7D%0D%0A%0D%0Ainterface%20ClassArray%20extends%20Array%3CClassValue%3E%20%7B%20%7D%0D%0A%0D%0Ainterface%20ClassNamesFn%20%7B%0D%0A%09(...classes%3A%20ClassValue%5B%5D)%3A%20string%3B%0D%0A%7D%0D%0A%0D%0Adeclare%20var%20classNames%3A%20ClassNamesFn%3B%0D%0A%0D%0A%0D%0A%0D%0A%0D%0A%0D%0Ainterface%20MyProps%20%7B%0D%0A%09foo%3F%3A%20boolean%3B%0D%0A%7D%0D%0A%0D%0Aconst%20myProps%3A%20MyProps%20%3D%20%7B%0D%0A%09%2F%2F%20nothing%0D%0A%7D%3B%0D%0A%0D%0Aconst%20classes%20%3D%20classNames(%7B%0D%0A%09%22some-class%22%3A%20myProps.foo%2C%0D%0A%7D)%0D%0A)